ozkour-back/google-api/slide.js

```diff
@@ -250,6 +250,32 @@ function test(auth, talks) {
           fields: "tableBorderFill",
         },
       },
+      {
+        updateTableColumnProperties : {
+          objectId: date.replaceAll("/", "-") + "-table",
```

Comment:
On pourrait definir l'object Id plus haut, et eviter qu'il soit calculer plusieurs fois
`const objectId = date.replaceAll("/", "-") + "-table";`
```suggestion
          objectId,
```

Reply:
c'est vrai, je m'occupe de changer ça